cards/src/main/resources/cards/custom/group2/set_633/verdant/weapon_verdant_quarterstaff.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Verdant Quarterstaff",
+  "baseManaCost": 3,
+  "type": "WEAPON",
+  "heroClass": "JADE",
+  "damage": 1,
+  "durability": 6,
+  "rarity": "COMMON",
+  "description": "Extra Strike",
+  "onEquip": {
```

Comment:
this should probably be done as an `AttributeAura` that does the `AURA_WINDFURY`